main_test.go

```diff
@@ -16,6 +16,7 @@ func TestParseConfig(t *testing.T) {
 	//
 	// Sets the environment to sane values
 	uri, user, password, logLevel, serverDomain := "URI", "user", "password", logrus.ErrorLevel, "server.com"
+	accountAPIAddress := "accounts"
```

Comment:
If we use a single var we'll need to add the port to it as well.

Reply:
This is just the test. The `uri` also doesn't have a valid uri because I just want to make sure we enforce setting it. If you provide an invalid value, then the initial ping will fail in `main()` and abort the startup.
Because at the time of parsing the address there is no way to verify that it works except for trying anyway.